volatility3/framework/constants/__init__.py

```diff
@@ -39,8 +39,8 @@
 
 # We use the SemVer 2.0.0 versioning scheme
 VERSION_MAJOR = 2  # Number of releases of the library with a breaking change
-VERSION_MINOR = 0  # Number of changes that only add to the interface
-VERSION_PATCH = 4  # Number of changes that do not change the interface
+VERSION_MINOR = 1  # Number of changes that only add to the interface
```

Comment:
The main tree is already up to 2.1.0, if this makes an additive change, please bump it to 2.2.0 (and update any plugins that rely on it).

Reply:
done